core/src/main/java/org/axonframework/eventsourcing/EventSourcingRepository.java

```diff
@@ -220,6 +219,17 @@ public EventSourcingRepository(AggregateFactory<T> aggregateFactory, EventStore
         return aggregate;
     }
 
+    /**
+     * Reads the events for the given aggregateIdentifier from the eventStore. This method can be overridden
```

Comment:
Nit: I'd rephrase the second phrase to `This method can be overridden to for example add filtering (by using the {@link DomainEventStream#filter(Predicate)} on the returned {@link DomainEventStream}.

Reply:
I wouldn't even mention the example, actually. We should try to avoid "steering" anyone in a specific direction as it may lead to confusion.
I would say "this method may be overridden to add pre or postprocessing to the loading of an event stream".